### ISSUE TITLE
Witness linear combination V2

### ISSUE THREAD

**Issue by StevenClontz** [TeamBasedInquiryLearning/checkit-tbil-la#31]
Something I just recalled from talking to Sharona at the Institute. She noticed that students were passing V2, but couldn't actually  take the RREF augmented matrix and use it to express the vector as a specific linear combination of the other vectors. So maybe this could be refactored in this format:

> Consider the set of vectors S=[SET].
> 1. For each of the below vectors, write a vector equation that must be satisfied for the vector to be a linear combation of vectors from S.
>     - [vector that belongs]
>     - [vector that doesn't] (randomly swap these)
> 2. Use these vector equations to explain why each of the above vectors is or is not a linear combination of vectors from S. If it is, give a specific example of a linear combination of vectors from S that equals it.

**Comment by siwelwerd** [comment]
Getting there... kind of want to disentangle things though. What about something like

1. Write a statement about solutions to a vector equation that is equivalent to the statement <true statement>
2. Explain why each of the following vectors is or is not a linear combination of [set of vectors]. 
3. Write the vector [vector] as a linear combination of vectors in [set].

**Comment by StevenClontz** [comment]
Gotcha, I think I can run with this, thanks!

**Comment by siwelwerd** [comment]
I think markdown ate a word. The idea with 1. was to always have a *true* statement to avoid the confusions in #19

**Comment by siwelwerd** [comment]
Let me try again for clarity:

Let S={[1 2 3] [4 5 6]} .
1. Write a statement about solutions to a vector equation that is equivalent to the statement "[5 7 9] is a linear combination of the vectors in S"
2. Explain why each of the following vectors is or is not a linear combination of vectors in S: [6 9 12] , [1 0 1]
3. Write the vector [3 3 3] as a linear combination of the vectors in S.

**Comment by StevenClontz** [comment]
When I sat down to implement this, I felt this got at the idea without feeling unnecessarily repetitive.

![image](https://user-images.githubusercontent.com/1559632/129254030-db901e80-1c01-400b-9d6f-69da43cfdbaa.png)
![image](https://user-images.githubusercontent.com/1559632/129254061-13c4a453-b30d-4e3a-a804-3651f22c90ba.png)

**Comment by siwelwerd** [comment]
Using a,b,c,d is significantly higher cognitive load.

**Comment by StevenClontz** [comment]
But without them, the sentence isn't actually used in the rest of the problem, breaking with the design of V3 et al.

But I think I have an idea that cuts the knot - I'll attempt it later tonight or tomorrow.

**Comment by StevenClontz** [comment]
This has the advantage of being a little shorter too.

![image](https://user-images.githubusercontent.com/1559632/129287345-5ed85cbc-4a13-4ab0-86a9-3e322824bda6.png)

**Comment by siwelwerd** [comment]
Not my favorite, but I don't have a great alternative (I liked what I proposed above but agree it's probably too long). Let's just roll with this for this semester and see.

Minor note: Do we want to say "Write a statement about a vector equation" or "Write a statement about the solutions of a vector equation"?  In the latter case, we could then maybe say "Solve the vector equations from part (a), and explain how these solutions determine if each vector is or is not a linear combination. (...)"

**Comment by StevenClontz** [comment]
I strongly prefer that the work students do in setting up an equivalent problem is then used to solve it, rather than re-rolling random vectors for each part, but agree that maybe there's a more clever way to achieve this. We'll see how it works in practice.

> "Write a statement about the solutions of a vector equation"? In the latter case, we could then maybe say "Solve the vector equations from part (a), and explain how these solutions determine if each vector is or is not a linear combination. (...)"

I like switching (a) to your suggestion the above, but leaving (b) as-is.
